tests/tests.py

```diff
@@ -1095,6 +1095,21 @@ def test_comments(self):
             comments = self.jira.comments(issue)
             assert len(comments) == 0
 
+    def test_expanded_comments(self):
+        comment1 = self.jira.add_comment(self.issue_1, "First comment")
+        comment2 = self.jira.add_comment(self.issue_1, "Second comment")
+        comments = self.jira.comments(self.issue_1, expand="renderedBody")
+        self.assertTrue(hasattr(comments[0], "renderedBody"))
+        ret_comment1 = self.jira.comment(self.issue_1, comment1.id,
+                                         expand="renderedBody")
+        self.assertTrue(hasattr(ret_comment1, "renderedBody"))
+        ret_comment2 = self.jira.comment(self.issue_1, comment2.id)
+        self.assertFalse(hasattr(ret_comment2, "renderedBody"))
+        comment1.delete()
```

Comment:
seems like this is a cleanup part. if this test fails on the jira cloud instance then it wont be cleaned up.
maybe delete any comments first on issue_1

Reply:
I presume you mean to move both self.assert calls down after the comment1.delete() and comment2.delete() calls, thus guaranteeing they get removed regardless of whether the asserts fail or not.

Reply:
Latest commit I've moved comment deletions to be before assert calls. Thanks for the review.

Reply:
Black failed